test/unit/defaultParams.js

```diff
@@ -11,7 +11,7 @@ describe('Default Parameter Tests', function () {
   before(function () {
     app = require('../../roosevelt')({
       appDir: path.join(__dirname, '../app/defaultParams'),
-      ignoreCLIFlags: true,
+      enableCLIFlags: false,
```

Comment:
When running through the test here for enableCLIFlags, it will say enableCLIFlags is false here when it is true in the defaults.json file. Is it supposed to be true or false by default?

Reply:
This line can be removed and the tests will all still pass